<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>679</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Version : 0.1 ‚Äì Local IPTV Player Interface with EPG, Favorites, and Filters</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLabel" name="titleLabel">
                        <property name="styleSheet">
                            <string>font-weight: bold; font-size: 16pt;</string>
                        </property>
                        <property name="text">
                            <string>Lecteur IPTV avec VLC</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignmentFlag::AlignCenter</set>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLineEdit" name="urlLineEdit">
                        <property name="placeholderText">
                            <string>Entrez URL du flux M3U ou chemin local</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout">
                        <item>
                            <widget class="QLineEdit" name="searchBar">
                                <property name="placeholderText">
                                    <string>üîç Rechercher une cha√Æne...</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="openFileButton">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property name="text">
                                    <string>Charger un fichier de cha√Æne...</string>
                                </property>
                                <property name="icon">
                                    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="clearSearchButton">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property name="toolTip">
                                    <string>Effacer le filtre</string>
                                </property>
                                <property name="text">
                                    <string>‚ùå</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QListWidget" name="channelsList"/>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
                        <item>
                            <widget class="QPushButton" name="playButton">
                                <property name="text">
                                    <string>‚ñ∂ Play</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pauseButton">
                                <property name="text">
                                    <string>‚è∏ Pause</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="stopButton">
                                <property name="text">
                                    <string>‚èπ Stop</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
  <layout class="QHBoxLayout" name="layoutVolume">
    <item>
      <widget class="QSlider" name="volumeSlider">
        <property name="orientation">
          <enum>Qt::Horizontal</enum>
        </property>
        <property name="minimum">
          <number>0</number>
        </property>
        <property name="maximum">
          <number>100</number>
        </property>
        <property name="value">
          <number>80</number>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QPushButton" name="muteButton">
        <property name="text">
          <string>üîá</string>
        </property>
        <property name="toolTip">
          <string>Couper / r√©activer le son</string>
        </property>
      </widget>
    </item>
  </layout>
</item>
<item>
  <widget class="QSlider" name="progressSlider">
    <property name="orientation">
      <enum>Qt::Horizontal</enum>
    </property>
  </widget>
</item>
                <item>
                    <widget class="QLabel" name="statusLabel">
                        <property name="text">
                            <string>Statut : Arr√™t√©</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QWidget" name="videoFrame" native="true">
                        <property name="minimumSize">
                            <size>
                                <width>640</width>
                                <height>360</height>
                            </size>
                        </property>
                        <property name="styleSheet">
                            <string>background-color: black;</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1000</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
